.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @holgiosalos @danielgalvis98 @kliver98 @AlejaGonzalezV @NicolasB2 @manuelq12 @Valeryibarra @veronicatofino
```

Comment:
Add EOF (End of File)

Reply:
In this way?
* @holgiosalos @danielgalvis98 @kliver98 @AlejaGonzalezV @NicolasB2 @manuelq12 @Valeryibarra @veronicatofino EOF